### DISCUSSION TITLE
Business Mediation Policy

### DISCUSSION THREAD

**Discussion by MattiSG** [OpenTermsArchive/terms-types#19] [Added]
### Google Platform-to-Business (P2B) Mediation Help Center

https://support.google.com/legal/answer/9792937

**Comment by MattiSG** [comment]
### Deliveroo's partner support policy

https://help.deliveroo.com/en/articles/4239076-deliveroo-s-partner-support-policy

**Comment by MattiSG** [comment]
### Amazon's Mediation Policy

https://sellercentral-europe.amazon.com/help/hub/reference/external/G67ETGRC3ZJQBTVT?ref=efph_G67ETGRC3ZJQBTVT_cont_521&locale=en-GB

**Comment by MattiSG** [comment]
## Recap of provided examples

All three listed examples provide the same basic information: who is the nominated mediator, eligibility criteria and process overview. 2 out of 3 mention explicitly the P2B regulation.

## Relevance

These documents implement a requirement of the EU P2B regulation ([article 12](https://eur-lex.europa.eu/eli/reg/2019/1150/oj#d1e1148-57-1)). We can thus reasonably expect an increasing number of platforms operating in the EU to provide such documents.

## Tryptich

- Writer: `intermediation service provider`
- Audience: `business users`
- Object: `eligibility and process of mediation after internal complaints handling failed`
- References:
  - Open Terms Archive discussion: https://github.com/ambanum/OpenTermsArchive/discussions/933
  - P2B Regulation 2019/1150, Article 12: https://eur-lex.europa.eu/eli/reg/2019/1150/oj#d1e1148-57-1

## Naming

### Intent

- This document describes a planned process more than it sets forth conditions, rights or duties. I thus believe it should be a “policy”.
- It targets only business users, not end users who might also have a right to mediation.
- It specifically focuses on mediation, not on other processes that might have failed before. If they were also included, the more generic type `Complaints Policy` would be used.

### Proposals

1. **Business Mediation Policy**

**Reply by lucielechardoy** [reply]
Thanks @MattiSG . As you rightly say, this mediation policy should be addressed to business users, and it is about solving issues between the platform and the business user (so no B2B or B2C dispute). Therefore I would suggest to specify this in the name and call it 'P2B mediation policy' or 'plaftorm-to-business mediation policy'

**Reply by MattiSG** [reply]
I would prefer that we avoid mentioning P2B in the document name unless it is strictly related as in https://github.com/ambanum/OpenTermsArchive/discussions/940, so we can use that document type if the provider offers a Business Mediation Policy outside of the EU, or if other jurisdictions mandate similar provisions. For example, we prefer to have `Privacy Policy` and `Trackers Policy` rather than `GDPR`/`CCPA`/`LGPD` document types 🙂

**Reply by lucielechardoy** [reply]
Here I didn't mean P2B as a legislation name but more as opposed to B2B or B2C (because B2C mediation between buyers and sellers is a requirement of another EU legislation for example). But if you prefer to keep the category a bit high level in case it can fit more cases in the future that's also fine

**Reply by MattiSG** [reply]
Thanks for your clarifications @lucielechardoy! I believe that for end users, we'd rather use the existing `Complaints Policy`.
We'll go with `Business Mediation Policy` then! 🙂

**Comment by MattiSG** [comment]
Added via https://github.com/ambanum/OpenTermsArchive/pull/941!
